### PR TITLE
[ja] add docs for owners-dependents

### LINKED ISSUE #38791: [ja] Translate concepts/overview/working-with-objects/owners-dependents/ into Japanese
**This is a Feature Request**

**What would you like to be added**

Translate https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/ into Japanese

**Why is this needed**

**Comments**

/language ja
/assign

### PR DESCRIPTION
**Pull request by FeLvi-zzz**
Add desrtiption of owners-dependents.
https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/

close #38791

### CODE REVIEW

File: content/ja/docs/concepts/overview/working-with-objects/owners-dependents.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+weight: 90
+---
+
+<!-- overview -->
+
+Kubernetesでは、いくつかのオブジェクトは他のオブジェクトの*オーナー*になっています。
+例えば、{{<glossary_tooltip text="ReplicaSet" term_id="replica-set">}}はPodの集合のオーナーです。
+これらの所有されているオブジェクトはオーナーに*従属*しています。
+
+オーナーシップはいくつかのリソースでも使われている[ラベルとセレクター](/docs/concepts/overview/working-with-objects/labels/)とは仕組みが異なります。
```
Comment (bells17) [inline content/ja/docs/concepts/overview/working-with-objects/owners-dependents.md:13]: `/ja/docs/concepts/overview/working-with-objects/labels/`

日本語ページがあるようです
https://kubernetes.io/ja/docs/concepts/overview/working-with-objects/labels/

File: content/ja/docs/concepts/overview/working-with-objects/owners-dependents.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## オーナーシップとファイナライザー
+
+Kubernetesでリソースを削除するとき、APIサーバーはリソースを管理するコントローラーに[ファイナライザールール](/docs/concepts/overview/working-with-objects/finalizers/)を処理させることができます。
+{{<glossary_tooltip text="ファイナライザー" term_id="finalizer">}}はクラスターが正しく機能するために必要なリソースを誤って削除してしまうことを防ぎます。
+例えば、まだPodが使用中の`PersistentVolume`を削除しようとするとき、`PersistentVolume`が持っている`kubernetes.io/pv-protection`ファイナライザーにより、削除は即座には行われません。
+
+その代わり、Kubernetesがファイナライザーを削除するまでボリュームは`Terminating`ステータスのまま残り、`PersistentVolume`がPodに利用されなくなった後で削除が行われます。
+
+またKubernetesは[フォアグラウンド、孤立したオブジェクトのカスケード削除](/docs/concepts/architecture/garbage-collection/#cascading-deletion)を行ったとき、オーナーリソースにファイナライザーを追加します。
```
Comment (bells17) [inline content/ja/docs/concepts/overview/working-with-objects/owners-dependents.md:56]: `/ja/docs/concepts/architecture/garbage-collection/#cascading-deletion`

日本語ページがあるようです
https://kubernetes.io/ja/docs/concepts/architecture/garbage-collection/#cascading-deletion

**Review by bells17 — COMMENTED** [review]
2点コメントしました。
ご確認ください。

**Review by t-inu — COMMENTED** [review]
Japanese finalizer page will be available after merging PR https://github.com/kubernetes/website/pull/38768.
So, how about wait for the merge, and then reflect my suggestions?

**Review by t-inu — COMMENTED** [review]
I also reviewed the contents.

**Comment by FeLvi-zzz** [comment]
> Japanese finalizer page will be available after merging PR https://github.com/kubernetes/website/pull/38768.
So, how about wait for the merge, and then reflect my suggestions?

I reflected your suggestions, so I wait for it.

File: content/ja/docs/concepts/overview/working-with-objects/owners-dependents.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+v1.20以降では、クラスタースコープの従属オブジェクトが名前空間付きのオブジェクトをオーナーとした場合、
+解決できないオーナーリファレンスを持っているものとして扱われ、ガベージコレクションの対象とすることができません。
+
+v1.20以降で、ガベージコレクターが無効な名前空間またぎの`ownerReference`や名前空間付きのオーナーに依存するクラスタースコープのオブジェクトなどを検知した場合、`OwnerRefInvalidNamespace`を理由としたEventを出し、`involvedObject`で無効な従属オブジェクトを報告します。
+`kubectl get events -A --field-selector=reason=OwnerRefInvalidNamespace`を実行することでこの種類のEventを確認することができます。
+{{< /note >}}
+
+## オーナーシップとファイナライザー
+
+Kubernetesでリソースを削除するとき、APIサーバーはリソースを管理するコントローラーに[ファイナライザールール](/docs/concepts/overview/working-with-objects/finalizers/)を処理させることができます。
```
[the remark this answers is not in the document]
Comment (t-inu) [reply content/ja/docs/concepts/overview/working-with-objects/owners-dependents.md:50]: I've checked the line 62 has been updated.
Are you going to update the line 50 later?

[the remark this answers is not in the document]
Comment (FeLvi-zzz) [reply content/ja/docs/concepts/overview/working-with-objects/owners-dependents.md:50]: Oh, I missed it...
I'm updating.

**Review by t-inu — APPROVED** [review]
/lgtm

**Comment by t-inu** [comment]
/hold
This PR contains a link to `/ja/docs/concepts/overview/working-with-objects/finalizers/`.
That is a new page and reviewing at https://github.com/kubernetes/website/pull/38768.
Feel free to unhold anyone when that PR is merged.

**Review by nasa9084 — COMMENTED** [review]
/assign
/approve

will `/unhold` after the relevant PR is merged

**Comment by t-inu** [comment]
PR https://github.com/kubernetes/website/pull/38768 was merged.
/unhold

**Pull request merged**